web/src/composebox_typeahead.js

```diff
@@ -1262,10 +1262,4 @@ export function initialize({on_enter_send}) {
     });
 
     initialize_compose_typeahead("textarea#compose-textarea");
-
-    $("#private_message_recipient").on("blur", function () {
-        const val = $(this).val();
-        const recipients = typeahead_helper.get_cleaned_pm_recipients(val);
-        $(this).val(recipients.join(", "));
-    });
```

Comment:
Just a comment on the commit message -- it's not relevant that the block was introduced 10 years ago -- most code introduced at that date does something important -- but more that it should have been removed when we migrated this input to use pills.

Reply:
That makes sense, I meant to mention it should have been removed when we moved to pills. Thanks for calling this out.